### ISSUE TITLE
 No response from server on slumbot.com

### ISSUE THREAD

**Issue by shiweixu97** [ericgjackson/slumbot2019#23]
Dear @ericgjackson

I developed a poker agent and try to evaluate it on slumbot.com. However I found something wrong on the website, showing that "no response from server on slumbot.com". Could you help solve this problem? Thanks!

**Comment by ericgjackson** [comment]
I'll take a look later today when I'm back home.  Thanks for letting me know.

**Comment by JbCourtois** [comment]
Same issue here.
- The "new_hand" endpoint works fine
- The "act" endpoint returns http 401 errors, even after logging in
- The "register" endpoint returns http 400 errors

**Comment by ericgjackson** [comment]
The site is up and running again.

**Comment by shiweixu97** [comment]
I wonder if i can peek slumbot's hands through API at the end of the game?

**Comment by JbCourtois** [comment]
Yes, the response at the end of of each end contains slightly more details:

`{"action":"b200c/kb200f","client_pos":0,"hole_cards":["Kd","2h"],"board":["Th","9s","4s"],"bot_hole_cards":["Kh","7h"],"winnings":-200,"won_pot":-600,"session_num_hands":19,"baseline_winnings":0,"session_total":8850,"session_baseline_total":4450}`

Slumbot's hand is in the attribute `bot_hole_cards`.

**Comment by shiweixu97** [comment]
> 

Thanks! It helps me a lot.
